### PR TITLE
Model builders refactoring: rename dirs, targets, file names

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
Changes in test builders/helpers:
    renamed files, folders, cmake targets ngraph -> ov, function -> model:
   
image

contrib PR: https://github.com/openvinotoolkit/openvino_contrib/pull/731
validation PR: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/2741

Namespace, includes refactoring will be in the next PR

### Tickets:
 - [CVS-111373]

### CODE REVIEW

File: src/common/low_precision_transformations/tests/CMakeLists.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ ov_add_test_target(
         gtest_main
         openvino::runtime::dev
         common_test_utils
-        lptNgraphFunctions
+        lpt_ov_models
```
Comment (ilya-lavrenov) [inline src/common/low_precision_transformations/tests/CMakeLists.txt:16]: maybe `ov_lpt_models`?

File: src/common/low_precision_transformations/tests/avg_pool_transformation.cpp [hunk 2]
```diff
@@ -13,8 +13,8 @@
 
 #include "common_test_utils/ov_test_utils.hpp"
 #include "layer_transformation.hpp"
-#include "lpt_ngraph_functions/avg_pool_function.hpp"
-#include "lpt_ngraph_functions/common/dequantization_operations.hpp"
+#include "lpt_ov_models/avg_pool_function.hpp"
```
Comment (ilya-lavrenov) [inline src/common/low_precision_transformations/tests/avg_pool_transformation.cpp:16]:
```suggestion
#include "lpt_ov_models/avg_pool.hpp"
```
?

File: src/common/low_precision_transformations/tests/CMakeLists.txt (hunk 1 above)
Comment (itikhono) [reply src/common/low_precision_transformations/tests/CMakeLists.txt:16]: done

File: src/common/low_precision_transformations/tests/avg_pool_transformation.cpp (hunk 2 above)
Comment (itikhono) [reply src/common/low_precision_transformations/tests/avg_pool_transformation.cpp]: done for all such cases

File: src/common/low_precision_transformations/tests/CMakeLists.txt (hunk 1 above)
Comment (ilya-lavrenov) [reply src/common/low_precision_transformations/tests/CMakeLists.txt:16]: let's rename the target itself :)

**Review by evkotov — APPROVED** [review]

**Review by mateusztabaka — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Review by v-Golubev — APPROVED** [review]

**Pull request merged**